src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/AssessmentTransformerTest.kt

```diff
@@ -239,4 +248,13 @@ class AssessmentTransformerTest {
     assertThat(apiSummary.risks).isEqualTo(risksTransformer.transformDomainToApi(personRisks, domainSummary.crn))
     assertThat(apiSummary.person).isNotNull
   }
+
+  companion object {
+    @JvmStatic
+    fun assessmentDecisionPairs(): Stream<Arguments> = Stream.of(
```

Comment:
This might be better put before the tests (assuming there's not a Kotlin/Java reason you can't) just to save the misdirection of having to look at the bottom of the file, especially as the `value` is set to a string value, so it's not immediately obvious it's referring to a function.

Reply:
Done